openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedTab/activity-feed-tab-new.less

```diff
@@ -0,0 +1,402 @@
+/*
+ *  Copyright 2023 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+@import url('../../../styles/variables.less');
+
+.feed-explore-heading {
+  background-color: @grey-1;
+}
+
+.activity-feed-tab {
+  display: flex;
+  gap: 20px;
+  width: 100%;
+  width: 100%;
+  overflow-x: hidden;
+
+  .custom-menu.ant-menu-root.ant-menu-inline {
+    .ant-menu-item {
+      height: 40px;
+    }
+  }
+
+  .center-container {
+    flex: 1;
+    min-width: 0;
+    height: @user-profile-page-panel-height;
+    overflow-y: scroll;
+    border-radius: 12px;
+    background-color: @white;
+    padding: 14px 20px 20px 20px;
+    .full-width {
+      flex: 10;
+    }
+  }
+  .right-container {
+    flex: 1;
+    min-width: 0;
+    border: none;
+    height: @user-profile-page-panel-height;
+    overflow-y: scroll;
+    background-color: @white;
+    border-radius: 12px;
+    .full-width {
+      flex: 0;
+    }
+
+    .activity-feed-card-container {
+      margin: 0px;
+    }
+
+    .activity-feed-comments-container {
+      border-radius: 12px;
+      padding: 16px;
+      margin: 20px 0px;
+      border: 0.8px solid #dfdfdf;
+      background: rgba(239, 244, 250, 0.25);
+    }
+
+    .activity-feed-reply-button {
+      border-radius: 24px;
+      border: 1px solid #d4d4d8;
+      color: #52525b;
+      font-size: 12px;
+    }
+    .comments-input-field {
+      background-color: #f4f5f7;
+      border-radius: 20px;
+      border: 1px solid #d4d4d8;
+      height: 36px;
+      font-size: 12px;
+      line-height: 14.52px;
+    }
+  }
+
+  .left-container {
+    flex: 0 0 @left-side-panel-width;
+    border-right: @global-border !important;
+  }
+  .activity-feed-card-new {
+    border-radius: 8px;
+    margin: 10px 0px;
+    background: #f5f5f5;
```

Comment:
Can we store colors in variable and use them? at least for the commonly used colors

Reply:
added variables for common colors